.travis.yml

```diff
@@ -1,30 +1,23 @@
 sudo: false
 language: python
-python:
-    - "2.7"
-    - "3.4"
```

Comment:
oh, weird! Can you just put a comment above this line then? Something like "we aren't explicitly declaring all versions needed by tox because ...". I think this is a weird assumption by Travis. A comment will prevent someone from trying to fix it later.